lithops/standalone/backends/aws_ec2/aws_ec2.py

```diff
@@ -160,7 +159,7 @@ def _delete_worker_vm_instances(self):
         if ins_to_delete:
             self.ec2_client.terminate_instances(InstanceIds=ins_to_delete)
 
-    def clean(self, delete_master=False, force=False):
+    def clean(self, delete_master=True, force=False):
```

Comment:
@aitorarjona In principle,[ to keep consistency between backends](https://github.com/lithops-cloud/lithops/tree/master/runtime/ibm_vpc#deploy-the-image), the master VM should only be deleted on `lithops clean --all`